wp-cache.php

```diff
@@ -3688,16 +3686,29 @@ function wpsc_enable_preload() {
 	wp_schedule_single_event( time() + 10, 'wp_cache_full_preload_hook' );
 }
 
+function wpsc_get_post_types() {
+
+	$preload_type_args = apply_filters( 'wpsc_preload_type_args', array(
```

Comment:
There's not much between them, but I'll go with `wpsc_preload_post_types_args` to look similar to the filter below it.